docs/sources/administration/database-encryption.md

```diff
@@ -1,8 +1,8 @@
 +++
-title = "Database encryption"
+aliases = ["", "/docs/grafana/latest/administration/database-encryption"]
```

Comment:
What's the behavior of this empty alias?

Reply:
I'm honestly not sure what Hugo would have done with this. It's a bug in the tool that has been fixed and the aliases updated in aa55c56d5f.